app/modules/import-export-customization/assets/js/import/pages/import-complete.js

```diff
@@ -0,0 +1,156 @@
+import { Box, Typography, Card, CardContent, Button, Link, Stack } from '@elementor/ui';
+import { useNavigate } from '@reach/router';
+import { __ } from '@wordpress/i18n';
+import { useEffect } from 'react';
+import { BaseLayout, TopBar, PageHeader, Footer } from '../../shared/components';
+import { useImportContext } from '../context/import-context';
+
+const Illustration = () => (
+	<Box
+		sx={ {
+			height: 140,
+			alignSelf: 'center',
+			borderRadius: 2,
+			display: 'flex',
+			alignItems: 'center',
+			justifyContent: 'center',
+		} }
+	>
+		<img
+			src={ elementorAppConfig.assets_url + 'images/kit-is-live.svg' }
+			alt="Kit is live illustration"
+		/>
+	</Box>
+);
+
+const ExternalLinkIcon = () => (
+	<Box
+		component="i"
+		sx={ { fontFamily: 'eicons' } }
+		className="eps-icon eicon-editor-external-link"
+	/>
+);
+
+export default function ImportComplete() {
+	const { isCompleted } = useImportContext();
+	const navigate = useNavigate();
+
+	const headerContent = (
+		<PageHeader title={ __( 'Import', 'elementor' ) } />
+	);
+
+	const footerContent = (
+		<Stack direction="row" spacing={ 1 }>
+			<Button
+				variant="outlined"
+				color="secondary"
+				startIcon={ <ExternalLinkIcon /> }
+				size="small"
+				data-testid="see-it-live-button"
+			>
+				{ __( 'See it Live', 'elementor' ) }
+			</Button>
+			<Button
+				variant="contained"
+				color="primary"
+				size="small"
+				sx={ { px: 4 } }
+				data-testid="close-button"
+			>
+				{ __( 'Close', 'elementor' ) }
+			</Button>
+		</Stack>
+	);
+
+	useEffect( () => {
+		if ( ! isCompleted ) {
+			navigate( '/import-customization', { replace: true } );
+		}
+	}, [ isCompleted, navigate ] );
```

Comment:
@max-zu Do we need to wrap the redirects in a `useEffect` hook? In the Export app we just used `<Redirect>` and checking the value from the context.